core/domain/suggestion_registry.py

```diff
@@ -283,11 +283,6 @@ def validate(self):
                     suggestion_models.SCORE_TYPE_CONTENT,
                     self.get_score_type()))
 
-        if self.get_score_sub_type() not in constants.ALL_CATEGORIES:
```

Comment:
Perhaps worth adding a comment to explain that the validation for the score sub-type matches the validation for the exploration category?

Also -- is there no validation at all for the exploration category? I thought there would be something basic at least, like a max length limit and alpha chars.